python/test/conftest.py

```diff
@@ -0,0 +1,6 @@
+# configuration file for pytest that applies to this folder and all subfolders
+
+def pytest_configure(config):
+    """hook that is run before executing any tests"""
+    import opendp.prelude as dp
+    dp.enable_features("rust-stack-trace")
```

Comment:
We don't want it enabled by default for users, because users can't follow these traces to sources. OTOH it's useful for debugging and I personally prefer it on all the time, because I get useful links that point to sources. However, some doctests fail when it's on, so I can't just enable it globally.